src/App.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import styled from "styled-components";
+import TimersView from "./views/TimersView";
+import CreateTimerView from "./views/CreateTimerView";
+import ButtonsView from "./views/ButtonsView";
+import { AppProvider } from "./AppProvider";
+import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
+
+const Container = styled.div`
```

Comment:
I really like the simplicity of your navigation - works well, has good feedback when something happens. great job!